packages/clay-empty-state/docs/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// eslint-disable-next-line
+import Editor from '$clayui.com/src/components/Editor';
```

Comment:
> * I've had to tell ESLint to ignore the import line because if I don't the image doesn't get properly loaded and yarn fails to compile the code saying imports are bad

So what is ESLint complaining about here? None of the other places that we do that kind of import seem to be causing problems ([random example](https://github.com/liferay/clay/blob/e62e074dedc299f4f0b42d10ed0074ab9e86d60f/packages/clay-date-picker/docs/index.js#L6))

(Aside: in general better to disable a specific lint rule on  instead of all rules on a line, because otherwise it's not clear what the problem you are suppressing is, and you might end up suppressing a separate, real problem that you don't intend to.)